src/Metrics.php

```diff
@@ -65,18 +76,22 @@ public function init()
         if ($this->metricsInterval > 0) {
             $this->registerInstance();
             $this->sendMetrics();
```

Comment:
I wonder if we need this extra send metrics call since we have the register shutdown one.

Reply:
it's redundant, will remove